Tools/autotest/arducopter.py

```diff
@@ -972,7 +971,7 @@ def setup_rc(mavproxy):
     mavproxy.send('rc 3 1000\n')
 
 
-def fly_ArduCopter(binary, viewerip=None, use_map=False, valgrind=False, gdb=False, frame=None, params=None, gdbserver=False):
+def fly_ArduCopter(binary, viewerip=None, use_map=False, valgrind=False, gdb=False, frame=None, params=None, gdbserver=False, speedup=20):
```

Comment:
I know @WickedShell is looking at increasing this to 20, but I'd prefer that change in a separate commit. Also, as I wrote in another comment, this default value won't actually matter.

Reply:
Leaving the 20x to the log download one is fine with me, I can reintroduce it then.

Reply:
@OXINARF Good call, it's not stable for me @ 20 - and I didn't actually mean for this to slip in!